### PR TITLE
GKE, VPC modules for GCP

### PR DESCRIPTION
**Pull request by pk0331**
GKE Modules for VPC, GKE

### CODE REVIEW

File: modules/gcp/gke/main.tf [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  horizontal_pod_autoscaling      = var.horizontal_pod_autoscaling
+  enable_vertical_pod_autoscaling = var.enable_vertical_pod_autoscaling
+  kubernetes_version              = var.kubernetes_version
+  initial_node_count              = var.initial_node_count ##How many instances should be launched in each zone
+  node_pools                      = var.node_pools
+  remove_default_node_pool        = var.remove_default_node_pool
+  cluster_resource_labels = merge(var.default_labels, var.labels)
+  //kubernetes_version = var.kubernetes_version
+  /*Node Pool taints, Labels, tags */
+  node_pools_labels         = var.node_pools_labels
```
Comment (PrashantRaj18198) [inline modules/gcp/gke/main.tf:51]: the `default_labels` should be applied to all the resources that support labels. We'll need to add `default_labels` to each `node_pool`.

File: modules/gcp/vpc/variables.tf [hunk 2]
```diff
@@ -0,0 +1,72 @@
+variable "project_id" {
```
Comment (PrashantRaj18198) [inline modules/gcp/vpc/variables.tf:1]: Does vpc not have resources that support `labels`?

**Review by PrashantRaj18198 — CHANGES_REQUESTED** [review]
Minor changes please 🙏

File: modules/gcp/gke/main.tf (hunk 1 above)
Comment (PrashantRaj18198) [reply modules/gcp/gke/main.tf:51]: already present in `cluster_resource_labels` so should not be required for `node_pools_labels`.

File: modules/gcp/vpc/variables.tf (hunk 2 above)
Comment (PrashantRaj18198) [reply modules/gcp/vpc/variables.tf:1]: Confirmed on call by @pk0331 that `labels` aren't present for vpc.

**Review by suryaoruganti — APPROVED** [review]

**Pull request merged**